diff -uraN lumina-1.6.1/src-qt5/OS-detect.pri omv-lumina-1.6.1/src-qt5/OS-detect.pri
--- lumina-1.6.1/src-qt5/OS-detect.pri	2021-10-02 21:05:49.000000000 +0200
+++ omv-lumina-1.6.1/src-qt5/OS-detect.pri	2021-11-19 16:30:13.222725064 +0100
@@ -99,7 +99,7 @@
   message( $$MSG )
 
   # Setup the dirs needed to find/load libraries
-  INCLUDEPATH +=$${PREFIX}/include 
+  #INCLUDEPATH +=$${PREFIX}/include 
   isEmpty(LIBPREFIX){ LIBPREFIX=$${PREFIX}/lib }
 
   # If the detailed install variables are not set - create them from the general vars
