--- lumina/src-qt5/core/lumina-desktop/defaults/luminaDesktop.conf.omv~	2016-08-19 00:54:34.953935195 +0200
+++ lumina/src-qt5/core/lumina-desktop/defaults/luminaDesktop.conf	2016-08-19 00:58:38.284856483 +0200
@@ -24,18 +24,24 @@ session_playlogoutaudio=true #[true/fals
 # *.desktop files provide better support for input formats, and are recommended
 #Note: the last "ifexists" entry has the highest priority for each session utility
 session_default_terminal_ifexists=xterm.desktop
+session_default_terminal_ifexists=qterminal.desktop
 session_default_terminal_ifexists=lumina-terminal.desktop
 session_default_filemanager=lumina-fm.desktop
 session_default_webbrowser_ifexists=chromium-browser.desktop
 session_default_webbrowser_ifexists=firefox.desktop
 session_default_webbrowser_ifexists=qupzilla.desktop
+session_default_email_ifexists=thunderbird.desktop
+session_default_email_ifexists=kmail.desktop
 session_default_email_ifexists=trojita.desktop
 
 #DEFAULT UTILITIES FOR INDIVIDUAL MIME TYPES
 # Format: mime_default_<mimetype>[_ifexists]=<*.desktop file>
+mime_default_text/*_ifexists=kwrite.desktop
 mime_default_text/*_ifexists=lumina-textedit.desktop
 mime_default_audio/*_ifexists=vlc.desktop
+mime_default_audio/*_ifexists=smplayer.desktop
 mime_default_video/*_ifexists=vlc.desktop
+mime_default_video/*_ifexists=smplayer.desktop
 mime_default_application/zip_ifexists=peazip.desktop
 mime_default_application/x-compressed-tar_ifexists=peazip.desktop
 mime_default_application/x-bzip-compressed-tar_ifexists=peazip.desktop
@@ -43,7 +49,9 @@ mime_default_application/x-lrzip-compres
 mime_default_application/x-lzma-compressed-tar_ifexists=peazip.desktop
 mime_default_application/x-xz-compressed-tar_ifexists=peazip.desktop
 mime_default_application/x-tar_ifexists=peazip.desktop
+mime_default_unknown/*=kwrite.desktop
 mime_default_unknown/*=lumina-textedit.desktop
+mime_default_application/x-shellscript=kwrite.desktop
 mime_default_application/x-shellscript=lumina-textedit.desktop
 
 #THEME SETTINGS
@@ -86,10 +94,10 @@ favorites_add_ifexists=firefox.desktop
 favorites_add_ifexists=chromium-browser.desktop
 favorites_add_ifexists=qupzilla.desktop
 favorites_add_ifexists=thunderbird.desktop
+favorites_add_ifexists=kmail.desktop
 favorites_add_ifexists=trojita.desktop
 favorites_add_ifexists=smplayer.desktop
 favorites_add_ifexists=vlc.desktop
-favorites_add_ifexists=pithos.desktop
 favorites_add_ifexists=~/Documents
 favorites_add_ifexists=~/Downloads
 favorites_add_ifexists=~/Pictures
